lib/classes/CLI.test.js

```diff
@@ -157,6 +157,34 @@ describe('CLI', () => {
     });
   });
 
+  describe('#isConfigIndependentRequest()', () => {
+    it('should not load config file if running "config credentials" command', () =>
+      runServerless({
+        cwd: fixtures.map.configTypeYml,
```

Comment:
Let's use `configInvalid` fixture, having that, we just need to confirm that `runServerless doesn't crash (no need to test any outcome)